### DISCUSSION TITLE
How to set hotkey `CMD+Left Mouse Button` in keymap?

### DISCUSSION THREAD

**Discussion by mkdr4** [zed-industries/zed#9544] [Feature Requests]
### Check for existing issues

- [x] Completed

### Describe the feature

Greetings! Can you tell me how to set the combination `CMD+LMB` for actions in keymap?

_No response_

**Comment by JosephTLyons** [comment]
I think that this is a deficiency currently in the key bindings - we aren't able to customize key bindings that involve mouse click. I thought we had an issue lying around on that, but can't find it at the moment ...

**Reply by mkdr4** [reply]
And when?

**Reply by JosephTLyons** [reply]
I'm not sure where this will land on the roadmap, but I did make an issue you can sub to to keep up to date with it.

https://github.com/zed-industries/zed/issues/10647

**Comment by meruiden** [comment]
Looking for this as well, my brain is wired to use the mouse buttons on the side of the mouse to go back and forward (pane::GoBack and pane::GoForward).

**Reply by meruiden** [reply]
for some reason, on macos, the sidebuttons of my MX master mouse (connected with BT) register as NSEventTypeSwipe instead of NSOtherMouseDown which is why navigation didnt work for me

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
